src/Altinn.Notifications.Core/Models/Orders/NotificationOrder.cs

```diff
@@ -102,4 +102,32 @@ public string Serialize()
                 Converters = { new JsonStringEnumConverter() }
             });
     }
+
+    /// <summary>
+    /// Try to parse a json string into the <see cref="NotificationOrder"/>
+    /// </summary>
+    public static bool TryParse(string input, out NotificationOrder value)
+    {
+        NotificationOrder? parsedOutput;
+        value = new NotificationOrder();
+
+        if (string.IsNullOrEmpty(input))
+        {
+            return false;
+        }
+
+        try
+        {
+            parsedOutput = Deserialize(input!);
+
+            value = parsedOutput!;
+            return value.Id != null;
```

Comment:
how many properties should we check to ensure it is a valid order that has been parsed?